includes/admin/tools/export/pdf-reports.php

```diff
@@ -116,12 +116,13 @@ function give_generate_pdf( $data ) {
 	// Set Custom Font to support various currencies.
 	$pdf->SetFont( apply_filters( 'give_pdf_custom_font', $custom_font ), $font_style, 12 );
 
-	$year       = date( 'Y' );
+	// Object for getting stats.
+	$donation_stats = new Give_Payment_Stats();
+
 	$give_forms = get_posts( array(
-		'post_type'      => 'give_forms',
-		'year'           => $year,
-		'posts_per_page' => - 1,
-		'supply_filter'  => false,
+		'post_type'        => 'give_forms',
+		'posts_per_page'   => - 1,
```

Comment:
@ravinderk should I replace it with `99999999999` for `posts_per_page`?

Reply:
@emgk no need